web/src/engine/websites/ReaperScansBR.ts

```diff
@@ -0,0 +1,101 @@
+import { Tags } from '../Tags';
+import icon from './ReaperScansBR.webp';
+import { Chapter, DecoratableMangaScraper, Page, type MangaPlugin, Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON, FetchRequest, FetchWindowScript } from './../FetchProvider';
+
+const apiUrl = 'https://api.reaperscans.net';
+
+type JSONNextMangaID = {
+    props: {
+        pageProps: {
+            data : { id : string }
+        }
+    }
+};
+
+type JSONNextManga = {
+    props: {
+        pageProps: {
+            series: {
+                chapters: { id: number, chapter_name: string, chapter_slug: string} []
+            }
+        }
+    }
+};
+
+type JSONChapters = {
+    content: {
+        images: string[]
+    }
+};
+
+type JSONMangas = {
+    data: { id: number, title: string, series_slug : string}[]
+}
+
+@Common.MangaCSS(/^https?:\/\/reaperscans\.net\/series\//, 'div.series-title > h1')
```

Comment:
This regex seems to also match [chapter links](https://reaperscans.net/series/hard-carry-support-1680818400091/capitulo-01), but should only match manga links

Reply:
Suggestion to ensure the regex always ends without any additional slash
`/^https?:\/\/reaperscans\.net\/series\/[^\/]+$/,`